.github/tests/combinations/fork9-cdk-erigon-validium.yml

```diff
@@ -7,9 +7,8 @@ args:
   zkevm_contracts_image: leovct/zkevm-contracts:v6.0.0-rc.1-fork.9-patch.1
   zkevm_prover_image: hermeznetwork/zkevm-prover:v6.0.8
   zkevm_node_image: hermeznetwork/zkevm-node:v0.7.3
-  cdk_validium_node_image: 0xpolygon/cdk-validium-node:0.7.0-cdk
+  cdk_validium_node_image: ghcr.io/0xpolygon/cdk-validium-node:0.6.4-cdk.10
   additional_services:
-    - pless_zkevm_node
```

Comment:
cc @praetoriansentry - we do not deploy legacy pless node in `fork9-cdk-erigon-validium` environments